1087 vertical alignment in contribute page

minump opened the pull request:
## Description
Contribute page styling updated.
Vertical alignment of radio buttons, text boxes and checkboxes are fixed.
Stylings fixed in contribute create and edit pages.

Styling for breadcrumbs yet to be updated (awaiting designer input for the same).

**Fixes #1087**

[comment]: # (This project only accepts pull requests related to open issues.)
[comment]: # (If suggesting a new feature or change, please discuss it in an issue first.)
[comment]: # (If fixing a bug, there should be an issue describing it with steps to reproduce.)

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Immediately
- [x] Within a week
- [ ] When possible

## Type of changes
_Please select a relevant option:_

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [x] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [x] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [x] I have updated the [CHANGELOG](../CHANGELOG.md).
- [ ] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [ ] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [ ] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [ ] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

---

[comment]: # (Template credit: This pull request template is based on Embedded Artistry {https://github.com/embeddedartistry/templates/blob/master/.github/PULL_REQUEST_TEMPLATE.md}, Clowder {https://github.com/clowder-framework/clowder/blob/develop/.github/PULL_REQUEST_TEMPLATE.md}, and TalAter {https://github.com/TalAter/open-source-templates} templates.)

sandeep-ps commented on contributions/catalog/webapps/templates/contribute/contribute.html:
```diff
@@ -143,24 +143,28 @@ <h2 id="contribute">Contribution Details</h2>
+                <div class="form-group">
+                    <label class="control-label col-sm-2">Contributor Details</label>
+                    <div class="col-sm-10">
+                        <button class="button button2" id="add-contributor-person" style="margin-left:0px">Add Person</button>
```
The margin-left style could be removed as without it, it's getting aligned well within the right side column. I'm unsure if everything needs to start on the same vertical line. Of course, we don't need elements to look moving on to the left side of the line.

minump replied:
updated. removed inline style.

sandeep-ps replied:
Thanks.

sandeep-ps commented:
The other vertical alignment changes look fine to me.

sandeep-ps commented:
As this task is about fixing vertical alignment issues, you may need to move the add environment variable button to the right side too. Thoughts? 
![image](https://user-images.githubusercontent.com/4306604/223533646-9f666284-e8e4-40bc-933d-23d0a4731ea6.png)

minump commented:
> As this task is about fixing vertical alignment issues, you may need to move the add environment variable button to the right side too. Thoughts? ![image](https://user-images.githubusercontent.com/4306604/223533646-9f666284-e8e4-40bc-933d-23d0a4731ea6.png)

If we have "Add environment variables" button in the middle of the form, it will need to move to the left (below the labels) when clicked, as the key-value inputs will need to take that space.

minump commented:
> As this task is about fixing vertical alignment issues, you may need to move the add environment variable button to the right side too. Thoughts? ![image](https://user-images.githubusercontent.com/4306604/223533646-9f666284-e8e4-40bc-933d-23d0a4731ea6.png)


Moved "Add environment variables" button to column divs.
Updated styling for add environment variables button.

minump commented:
Ready for review @sandeep-ps

sandeep-ps reviewed (approved):
The changes look good to me. I tested this PR by adding and editing a contribution. Thanks.

minump commented:
Merging to develop
